app/src/main/java/bg/zahov/app/data/model/Language.kt

```diff
@@ -1,17 +1,17 @@
 package bg.zahov.app.data.model
 
-class LanguageKeys {
-    companion object {
-        const val BULGARIAN = "Bulgarian"
-        const val ENGLISH = "English"
-    }
-}
+
 enum class Language(val key: String) {
-    Bulgarian(LanguageKeys.BULGARIAN),
-    English(LanguageKeys.ENGLISH);
+    Bulgarian("Bulgarian"),
+    English("English");
 
     companion object {
-        fun fromKey(key: String) = values().firstOrNull { it.key == key }.toString()
+        fun findByKey(key: String): Language {
```

Comment:
I think it's a better practice to use the `.entries` calls directly instead of those companion functions. It's easier to read and you don't need to add them n times to every enum class.

Reply:
changed

Reply:
Still present in [SetType.kt](https://github.com/HauntedMilkshake/fitness_app/pull/20/files#diff-74d82697f72c0933f32659c4e78e43f09d818741c7e074b154ba8db2da11fd04)

Reply:
@Gotalicp I think this is the last change needed here ☝️

Reply:
dont know where it is used but now it is fixed